macros/get_query.sql

```diff
@@ -28,41 +28,107 @@
 {#- For campaign level reports and lower, add campaign_fields -#}
 {%- if report_type in ['campaign', 'ad_group', 'ad', 'url', 'keyword', 'search'] -%}
     {%- for campaign_field in campaign_fields -%}
+        {#- When campaign_passthrough_metrics are defined, add them too but only to the ad_group report_type -#}
+        {%- if report_type == 'campaign' and var('ad_reporting__campaign_passthrough_metrics') -%}
```

Comment:
I worry that if we don't provide a default value for the variable in line then we could wind up with an error like the following for users.

![image](https://user-images.githubusercontent.com/74217849/235470578-5c2de509-925d-49b7-a15a-9832b1874043.png)

Providing a default value for the variable (like the below being an empty array) ensures that on any compilation state, dbt will not throw an error as there should always be a value for the variable regardless if the users sets it in their project.yml or does not.
```suggestion
        {%- if report_type == 'campaign' and var('ad_reporting__campaign_passthrough_metrics',[]) -%}
```

Reply:
Could we also apply the same update to the other variable calls in this macro. Just to be safe.

Reply:
Thanks a lot for this! I wasn't aware that this could happen, amended!